### ISSUE TITLE
Mark as damaged toggle (poster icon color turns red on map)

### ISSUE THREAD

**Issue by nscharioth** [verdigado/gruene_app#286]

**Comment by JuliaFreitagBGSt** [comment]
Issue is reviewed.

Note: the information about a damaged poster needs to be saved in the database (Grüne Atlas).

@nscharioth @leagerndt

**Comment by nscharioth** [comment]
> Issue is reviewed.
> 
> Note: the information about a damaged poster needs to be saved in the database (Grüne Atlas).
> 
> [@nscharioth](https://github.com/nscharioth) [@leagerndt](https://github.com/leagerndt)

Yes, that information will not be lost. It is saved via the API and can accessed for such functions in the cockpit.
